### PR TITLE
0.9.10 release

### PR DESCRIPTION
**Pull request by czue**
https://github.com/dimagi/jsonobject/issues/178

this look right to you @dannyroberts ?

anything else to do, or I can just publish to pypi once reviewed/merged?

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -59,6 +58,9 @@
         'Programming Language :: Python :: 2.7',
         'Programming Language :: Python :: 3',
         'Programming Language :: Python :: 3.5',
-        'Programming Language :: Python :: 3.6'
+        'Programming Language :: Python :: 3.6',
+        'Programming Language :: Python :: 3.7',
+        'Programming Language :: Python :: 3.8',
```
Comment (dannyroberts) [inline setup.py:63]: Nit: this probably works, but it's not one of the targets in our travis tests

**Review by dannyroberts — APPROVED** [review]
Yeah I think this is it. You can publish to pypi once it's merged

File: setup.py (hunk 1 above)
Comment (czue) [reply setup.py:63]: would be very surprising if it didn't work. do you think it's worth adding 3.8 to travis?

Comment (dannyroberts) [reply setup.py:63]: Yeah sure. These tests are very fast and run infrequently so it's unlikely to hurt our travis job credits or whatever

Comment (czue) [reply setup.py:63]: https://github.com/dimagi/jsonobject/pull/180

**Pull request merged**